distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -116,7 +140,8 @@ type File
                  file.with_output_stream [File_Access.Create, File_Access.Write] action
     with_output_stream : Vector File_Access -> (Output_Stream -> Any ! File_Error) -> Any ! File_Error
     with_output_stream self open_options action =
-        Managed_Resource.bracket (self.new_output_stream open_options) (_.close) action
+        if Context.Output.is_enabled.not then Error.throw (Forbidden_Operation.Error "File writing is forbidden as the Output context is disabled") else
+            Managed_Resource.bracket (self.new_output_stream open_options) (_.close) action
```

Comment:
Shouldn't this check be done in `new_output_stream` instead?

It feels like it allows us to circumvent the execution context if we use that instead of `with_output_stream` and I don't think it's the desired behaviour (if we want to circumvent the EC, we probably need to use the `Context.Output.with_enabled` instead of weird workarounds).